recipe/meta.yaml

```
@@ -12,63 +12,45 @@ source:
     - fix-doc-path.patch  # [build_platform != target_platform]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - {{ cdt('mesa-dri-drivers') }}  # [linux]
-    - {{ cdt('libselinux') }}  # [linux]
-    - {{ cdt('libxdamage') }}  # [linux]
-    - {{ cdt('libxfixes') }}  # [linux]
-    - {{ cdt('libxxf86vm') }}  # [linux]
-    - {{ cdt('expat') }}  # [linux]
-    - {{ cdt('libxext') }}  # [linux]
-    - {{ cdt('libxrender') }}  # [linux]
-    - {{ cdt('libxt') }}  # [linux]
-    - {{ cdt('libx11-devel') }}  # [linux]
-    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
-    - libxcb  # [linux]
+
     - pkg-config
     - gnuconfig  # [unix]
     - make  # [unix]
+    
+    - xorg-libx11  # [(build_platform != target_platform) and linux]
```

Can you add a comment like
```suggestion
    # These are needed since cross compiling builds need to build for the
    # native platform because ....
    - xorg-libx11  # [(build_platform != target_platform) and linux]
```

---

Typically it is quite rare to require these dependencies in the build section. please confirm they are required.